portfolio/src/main/webapp/index.html

```diff
@@ -180,6 +180,12 @@ <h2>Latest Reports</h2>
         <button onclick="route()">GO SAFE!</button>
     </div>
 
+    <div id="repeatedMarkerUI">
+        <h2>Sorry :(</h2>
```

Comment:
provide a better message on why this is happening.

Reply:
Done!